### DISCUSSION TITLE
How do other programming languages connect to this interface, such as C#

### DISCUSSION THREAD

**Discussion by zhuya1996** [miguelgrinberg/Flask-SocketIO#1819] [Q&A]
How do other programming languages connect to this interface, such as C#

**Comment by miguelgrinberg** [comment]
You need to use a socket.io client library for your language.

**Comment by zhuya1996** [comment]
C# Java and C + + all have this socket IO library




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Miguel ***@***.***&gt;; 
发送时间: 2022年4月25日(星期一) 下午2:25
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [miguelgrinberg/Flask-SocketIO] How do other programming languages connect to this interface, such as C# (Discussion #1819)






您需要为自己的语言使用一个 socket . io 客户端库。

—
直接回复此邮件，在 GitHub 上查看或取消订阅.
您收到这个消息是因为您编写了这个线程。留言 ID ***@***.*** need to use a socket.io client library for your language..—
